### ISSUE TITLE
 Remote refs should be removed [unpkg.com] -> issue in spectral

### ISSUE THREAD

**Issue by ioggstream** [italia/api-oas-checker#115]
## I expect

no remote refs 

## Instead

```
Access to fetch at 'https://unpkg.com/@stoplight/spectral@5.7.2/rulesets/oas/functions/refSiblings.js' from origin 'https://deploy-preview-98--api-validator.netlify.app' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
```

**Comment by emilioSp** [comment]
The ref is fully managed by the spectral engine on ruleset loading and it is triggered by this [line](https://github.com/italia/api-oas-checker/blob/master/rules/rules-template.yml.template#L6)

**Comment by ioggstream** [comment]
@emilioSp can you pls ask to stoplight/spectral? Do we have this file locally?

**Comment by ioggstream** [comment]
fixed via spectral update.
